Devs-AA/Barefoot
# #168662046 Build logout functionality

#### What does this PR do?
IT Build logout functionality 
#### Description of Task to be completed?
None
#### How should this be manually tested?
1. ```git clone https://github.com/Devs-AA/Barefoot.git```  and checkout into ft-logout-168662046 branch
2. run `npm install` and create  a `.env file` with the key value pair as specified on the `.env.example` file
3. run `npm run migrate:dev` and `npm run seed:dev`
4. Visit the route `http://localhost:3300/api/v1/users/auth/login` and use the following details to login 
```
email: requester1@gmail.com
password: Password1$
```
5. Get the token and use it in the header as `Bearer token` and visit the route: `http://localhost:3300/api/v1/users/auth/logout` with a DELETE verb . You should be successfully logout.

6. To confirm you have been succesfully logout visit any protected route such as `http://localhost:3300/api/v1/users/myaccount` you shouldn't be able to get to that route
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[#168662046](https://www.pivotaltracker.com/story/show/#168662046)

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/38135488/65821355-0ad9e000-e22c-11e9-919d-9468d2e17e7c.png)


Test coverage:

![image](https://user-images.githubusercontent.com/38135488/65821283-6fe10600-e22b-11e9-8fb2-2bdad2e6bcf2.png)




#### Questions:
